core/templates/pages/exploration-editor-page/settings-tab/settings-tab.component.html

```diff
@@ -690,15 +690,21 @@ <h3>
     </div>
   </mat-card>
 
-  <div *ngIf="canShowExploreVersionHistoryButton()" class="col-lg-12 col-md-12 col-sm-12">
-    <div class="mt-4">
-      <i>Last changed by <strong>{{ getLastEditedCommitterUsername() }}</strong> at <strong>version {{ getLastEditedVersionNumber() + 1 }}</strong>.</i>
-      <a class="text-primary explore-version-history-button" (click)="onClickExploreVersionHistoryButton()"> More details.</a>
+  <mat-card class="oppia-settings-card-container oppia-editor-card">
+    <div *ngIf="canShowExploreVersionHistoryButton(); else elseBlock">
```

Comment:
Prefer just having two ngIfs here, with opposing conditions. I don't think we use "else" or ng-templates much in other places in the codebase...

Reply:
Done.